avocado/utils/kernel.py

```diff
@@ -72,7 +72,8 @@ def download(self):
         full_url = self.URL + self.SOURCE.format(version=self.version)
         self.asset_path = asset.Asset(full_url, asset_hash=None,
                                       algorithm=None, locations=None,
-                                      cache_dirs=self.data_dirs).fetch()
+                                      cache_dirs=[self.work_dir], expire=0).fetch()
```

Comment:
NP, I have no strong opinions on that (other than the "library default cache dir" idea).